app/v1/connectors/BaseDesConnector.scala

```diff
@@ -35,6 +35,12 @@ trait BaseDesConnector {
     hc.copy(authorization = Some(Authorization(s"Bearer ${appConfig.desToken}")))
       .withExtraHeaders("Environment" -> appConfig.desEnv)
 
+  private[connectors] def desPostHeaderCarrier(implicit hc: HeaderCarrier): HeaderCarrier =
+    hc.copy(authorization = Some(Authorization(s"Bearer ${appConfig.desToken}")))
+      .withExtraHeaders("Environment" -> appConfig.desEnv,
+        "Accept" -> "application/json",
+        "OriginatorID" -> "MDTP")
```

Comment:
should OriginatorID be added to the GET too? old vat adds to GET and POST, uk.gov.hmrc.vatapi.connectors.BaseConnector L#37